### ISSUE TITLE
mongoose/mongodb template?

### ISSUE THREAD

**Issue by ralyodio** [robrighter/node-boilerplate#14]
Would it be possible to add support for a simple mongoose/mongodb setup?

**Comment by robrighter** [comment]
@chovy are you still interested in this feature? I am debating if I should add it in version2. I have the code pretty much written, but the issue is how to make it easy for people to get a mongo db setup. The goal for this project has always been to have everything work after the initial script is run. Setting up a mongo install might complicate things and be a barrier for new users.

**Comment by ralyodio** [comment]
Personally, I think it would be beneficial. What good is an app without a database instance? :-D

**Comment by ralyodio** [comment]
woops, I closed this thread on accident.

**Comment by kristoferjoseph** [comment]
+1 mongodb template

**Comment by crawfordcomeaux** [comment]
+1 for me, too

**Comment by mrjcleaver** [comment]
+1

**Comment by dgoujard** [comment]
+1 for me too

**Comment by malachaifrazier** [comment]
- 1 Ditto

**Comment by cyrilic** [comment]
+1

**Comment by davidmarkclements** [comment]
I disagree, this locks you into one database type (what if you want couchdb, or need mysql for legacy?)

 there are some cases you don't want a database instance, I started using expressling and found dependence on mongo to be an obstacle, and not an easy one to remove either as its integrated quite tightly.

The reason i'm now going to use node boilerplate is exactly because it doesn't have a db tie in.

Perhaps have db support in a seperate branch, one for mongo, then poss one for redis/couch/mysqll etc?

**Comment by niftylettuce** [comment]
once i fix up expressling you can use it http://expressling.com

**Comment by tchalvak** [comment]
Does anyone have a fork set to use mongodb to try out?  Err, I guess robrighter might.  I'd be interested in checking it out, even if it was just in a branch or fork of this repo.
